inference-engine/samples/benchmark_app/benchmark_app.hpp

```diff
@@ -237,4 +262,7 @@ static void showUsage() {
     std::cout << "    -load_config              " << load_config_message << std::endl;
 #endif
     std::cout << "    -qb                       " << gna_qb_message << std::endl;
+    std::cout << "    -ip                          <value>     "   << inputs_precision_message     << std::endl;
+    std::cout << "    -op                          <value>     "   << outputs_precision_message    << std::endl;
+    std::cout << "    -iop                        \"<value>\"    "   << iop_message                << std::endl;
```

Comment:
Do we want to flush each time?
```
Use of std::endl in place of '\n', encouraged by some sources, may significantly degrade output performance.
```
source :  https://en.cppreference.com/w/cpp/io/manip/endl